### ISSUE TITLE
Consider removing "Location Services" screen

### ISSUE THREAD

**Issue by matchboxbananasynergy** [GrapheneOS/platform_packages_apps_SetupWizard2#13]
This was added purely because it was in the legacy SetupWizard to avoid any potential loss in functionality.

That said, it is unclear if this screen is that useful considering that location is enabled by default, and one can disable it right after setup via the dedicated quick tile for it.

Especially because there are plans to add more screens to SetupWizard2, it makes sense to carefully think about useless/redundant screens to minimize setup time and ensure all screens have a real use.

**Comment by jiteshsingh** [comment]
makes sense to remove the screen if location services are already disabled by default.

**Comment by andromxda** [comment]
This can be closed since the location services screen is clearly being maintained and has recently received a big update in #45?
